### PR TITLE
Only show recently updated players in results. Number of days can be …

### PR DESCRIPTION
**Pull request by jteague9**
…configured in admin panel under constance.

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -16,6 +18,13 @@ def default(self, obj):
         return super().default(obj)
 
 
+def age_filter(days=config.AGE_LIMIT):
+    print(days)
```
Comment (tso) [inline app/views.py]: tsk

File: app/views.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from constance import config
+from datetime import datetime, timedelta
```
Comment (tso) [inline app/views.py]: import this from django i think to properly do timezone etc or smt

**Review by tso — APPROVED** [review]

File: app/views.py (hunk 2 above)
Comment (jteague9) [reply app/views.py]: Ok changed to `from django.utils import timezone`

File: app/views.py (hunk 1 above)
Comment (jteague9) [reply app/views.py]: Removed the print statement

**Pull request merged**